man/git-standup.md

```diff
@@ -25,10 +25,14 @@ The depth of recursive directory search. Defaults to 1.
 
 Enable the inclusion of symbolic links in recursive directory search.
 
--d 1
+-d days-ago
 
 The start of commit history. Defaults to 1, means "1 days ago".
 
+-w <weekstart-weekend>
+
+Limit the search range to weekdays. If weekstart and/or weekend are missing they default to Mon and Fri.
```

Comment:
Let's also doc the behavior in https://github.com/tj/git-extras/blob/d2a09c1753cbc27293bbeeada3e2697e359323f5/bin/git-standup#L99